src/pages/ExamplePage.tsx

```diff
@@ -28,34 +30,48 @@ export function ExamplePage() {
       return { enabled };
     }
 
-    setStarted(true);
-    setFinished(false);
-    // Stream the completions. Each element is the next stream chunk.
-    const stream = llms.openai.streamChatCompletions({
-      model: 'gpt-3.5-turbo',
-      messages: [
-        { role: 'system', content: 'You are a cynical assistant.' },
-        { role: 'user', content: message },
-      ],
-    }).pipe(
-      // Accumulate the stream content into a stream of strings, where each
-      // element contains the accumulated message so far.
-      llms.openai.accumulateContent(),
-      // The stream is just a regular Observable, so we can use standard rxjs
-      // functionality to update state, e.g. recording when the stream
-      // has completed.
-      // The operator decision tree on the rxjs website is a useful resource:
-      // https://rxjs.dev/operator-decision-tree.
-      finalize(() => {
-        setStarted(false);
-        setFinished(true);
-      })
-    );
-    // Subscribe to the stream and update the state for each returned value.
-    return {
-      enabled,
-      stream: stream.subscribe(setReply),
-    };
+    if (!useStream) {
+      // Make a single request to the LLM.
+      const response = await llms.openai.chatCompletions({
+        model: 'gpt-3.5-turbo',
+        messages: [
+          { role: 'system', content: 'You are a cynical assistant.' },
+          { role: 'user', content: message },
+        ],
+      });
```

Comment:
You could still set `setStarted` and `setFinished` before and after the request, for consistency 👍